TSC1/README.md

```diff
@@ -75,22 +75,27 @@ At least one week before the meeting:
     4. Copy action items from the previous meeting into the new agenda for follow up.
     5. If any items from the previous meeting agenda require more discussion, consider adding them to the current meeting or the parking lot.
     6. Look in the [parking lot](https://docs.google.com/spreadsheets/d/15haxZxYvl6BcaPLGw-3zctzUQad8F3hCYqsnD5ylqdo/edit#gid=0) for potential agenda items.
-    7. Also check for any agenda item requests in the above forum thread. If something from the thread doesn’t make it into the meeting, put it in the parking lot.
-    8. For substantive agenda items:
+    7. For substantive agenda items:
         1. list the goal(s) of the discussion and any things participants should do to prepare for it.
         2. Check in with the presenter to ensure they are ready to present and get a feel for what the discussion will consist of. This ensures you can facilitate it well.
+2. Get the URL for the meeting time from timeanddate.com:
+    1. Go to [this page](https://www.timeanddate.com/worldclock/fixedtime.html?msg=ODK+TSC+Meeting&iso=20190101T00&p1=1440&ah=1).
+    1. Click 'Modify Event'.
+    1. Fill in the actual meeting date and time.
+    1. Click 'Show Result'.
+    1. Copy the resulting URL.
 2. Create a thread for the meeting on the [forum](https://forum.opendatakit.org/).
```

Comment:
This should now be `3.`. Or just change all the numbers to `1. ` and let Markdown do the work.